spec/lib/pretty_text_spec.rb

```diff
@@ -731,6 +731,26 @@ def cook(*args)
       expect(cooked).to eq(html.strip)
     end
 
+    it "strips out unicode bidirectional (bidi) override characters and replaces with a highlighted span" do
+      cooked = <<~COOKED
+        <p>X</p>
+        <pre><code class="lang-auto">var isAdmin = false;
+        ‮
+        </code></pre>
```

Comment:
Can you use a ruby-escaped version of this character (characters?) so that the test is easier to work with? (and so that the editors/GH doesn't show the warnings 😃)